### PR TITLE
Update .travis.yml

### PR DESCRIPTION
**Pull request by mforets**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -27,13 +27,24 @@ branches:
   - /^testing-.*$/  # testing branches
   - /^v[0-9]+\.[0-9]+\.[0-9]+$/  # version tags
 
+matrix:
+  include:
+    - julia: 1.0
+      env: TEST_SUITE='using Pkg; Pkg.build("LazySets"); Pkg.test("LazySets"; coverage=true); Pkg.add(PackageSpec(name="ModelingToolkit", version="3.10.2"))'; DOCS=false
```
Comment (schillic) [inline .travis.yml:33]:
```suggestion
      env: TEST_SUITE='using Pkg; Pkg.build("LazySets"); Pkg.test("LazySets"; coverage=true)'; DOCS=false
```

**Comment by schillic** [comment]
`ModelingToolkit` has a new release and now it seems to work again (saw it in a different PR) :roll_eyes:
We can still merge here for stability reasons but maybe we should use the new version already?

**Comment by mforets** [comment]
> We can still merge here for stability reasons but maybe we should use the new version already?

i can update it later (with other changes, probably)

**Comment by schillic** [comment]
I think #2234 is the better solution.

**Pull request closed without merging**